concat-css.php

```diff
@@ -161,7 +161,7 @@ function do_items( $handles = false, $group = false ) {
 						return $path;
 					}, $css );
 					$mtime = max( array_map( 'filemtime', $paths ) );
-					$path_str = implode( $css, ',' ) . "?m={$mtime}";
+					$path_str = implode( ',', $css ) . "?m={$mtime}";
```

Comment:
nice catch! Just saw the alternative argument order is finally a [deprecated thing](https://www.php.net/manual/en/migration74.deprecated.php#migration74.deprecated.core.implode-reverse-parameters) in php 7.4.